### PR TITLE
435 Newer 1.6 fabric changes

### PR DESCRIPTION
**Pull request by mikeln**
Update to required newer fabric setup.

### CODE REVIEW

**Comment by mikeln** [comment]
NOTE: there is one RBAC piece to allow this to function correctly in 1.6 with rbac on.   however, that will be in the rbac PR coming up next.

**Comment by mikeln** [comment]
there is also another issue here.   
The config must specify the newer versions of the images:

But we probably don't want those versions for pre 1.6 kube clusters...  how to handle this?

Needed config setup:
```
 containers:
        kubePolicyController:
            version: v0.5.4
            location: calico/kube-policy-controller
          etcd:
            version: v3.1.5
            location: quay.io/coreos/etcd
          calicoCni:
            version: v1.8.3
            location: quay.io/calico/cni
          calicoNode:
            version: v1.1.3
            location: quay.io/calico/node
          flannel:
            version: v0.7.1
            location: quay.io/coreos/flannel
```

**Comment by coffeepac** [comment]
@mikeln did you test the new versions of the containers with a pre 1.6 k8s cluster?

**Comment by mikeln** [comment]
no, I did not run pre1.6 clusters with this fabric...

File: ansible/roles/kraken.nodePool/kraken.master.docker/templates/v1.6/controller-manager-manifest.yaml.jinja2 [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    image: {{ node.kubeConfig.hyperkubeLocation }}:{{ node.kubeConfig.version }}
+    command:
+    - /hyperkube
+    - controller-manager
+    - --master=http://127.0.0.1:8080
+    - --leader-elect=true
+    - --cloud-provider={{ cluster.providerConfig.provider }}
+    - --service-account-private-key-file=/etc/kubernetes/ssl/service-account-key.pem
+    - --root-ca-file=/etc/kubernetes/ssl/ca.pem
+    - --cluster-cidr={{ fabricConfigVal.options.network.network }}
```
Comment (mikeln) [inline ansible/roles/kraken.nodePool/kraken.master.docker/templates/v1.6/controller-manager-manifest.yaml.jinja2]: This file was added to this PR due to this line.   The new fabric requires that the master include the `--cluster-cid`r art for the master to work correctly.

**Comment by mikeln** [comment]
please retest this

**Comment by mikeln** [comment]
restest this please

**Comment by mikeln** [comment]
sorry, wanted to squash the commits.

**Pull request merged**